packages/ui/package.json

```diff
@@ -65,12 +65,15 @@
     "@open-wc/dedupe-mixin": "^1.4.0",
     "@open-wc/scoped-elements": "^3.0.5",
     "@popperjs/core": "^2.11.8",
-    "autosize": "^6.0.1",
+    "autosize": "^6.0.0",
     "awesome-phonenumber": "^6.4.0",
     "ibantools": "^4.3.9",
     "lit": "^3.1.2",
     "singleton-manager": "^1.7.0"
   },
+  "devDependencies": {
+    "@types/autosize": "^4.0.3"
```

Comment:
Should this be here, or with the other devDeps in the root `package.json`?

Reply:
Maybe the root is better indeed